packages/node_modules/pouchdb-collate/src/index.js

```diff
@@ -20,14 +20,11 @@ function collate(a, b) {
   if ((ai - bi) !== 0) {
     return ai - bi;
   }
-  if (a === null) {
-    return 0;
-  }
   switch (typeof a) {
     case 'number':
       return a - b;
     case 'boolean':
-      return a === b ? 0 : (a < b ? -1 : 1);
+      return a < b ? -1 : 1;
```

Comment:
this code path was impossible to hit because if two booleans are identical, then collate will return 0 above during the `a === b` test